e2e-tests/cypress/integration/tests/periodtickets.e2e_spec.js

```diff
@@ -20,13 +20,17 @@ describe('period ticket purchase', () => {
     });
 
     it('should display default ticket parameters', () => {
+        const defaultPeriodProduct = '24-timersbillett'
+        const defaultPeriodProductPrice = '126,00'
+        const defaultPeriodProductMva = '15,12'
+
         //Type
         newTicket.travelType().should("contain", "Buss og trikk")
 
         //Product
         newTicket.productsSection().then($product => {
             options.areVisible($product, true)
-            options.value($product).should("contain", "30-dagersbillett")
+            options.value($product).should("contain", defaultPeriodProduct)
```

Comment:
Kan være det hadde vært enklere med en sjekk om 30-dagersbillett eksisterte f.eks, kan bli litt sårbart dersom man endrer på sorteringen. Med mindre det da etableres at 24-timers skal være den første. Det kan jo være det er hensiktsmessig å sortere etter varighet, men det kan og være det er lurt å ta "det som blir brukt mest".

Reply:
Ja, denne testen er ment å sjekke hva som er default - dvs den vi velger at skal være forhåndsvalgt. En sjekk av remote config på en måte. Jeg vil ikke tro at 24-timers er den som bør stå som forhåndsvalgt. Her vil jeg tro Hilde&co (evt marked) har noe å si.

Har i tillegg lagt ved en test (ny commit) som sjekker at alle tilgjengelige produkter er til stede - og ingen fler.

Reply:
Så det er en test for å sikre at vi har et forhold til sortering / forhåndsvalgt verdi